Tests/js/apps/assets/ez-platformuiapp-tests.js

```diff
@@ -1477,19 +1515,52 @@ YUI.add('ez-platformuiapp-tests', function (Y) {
                 method: 'set',
                 args: ['id', undefined]
             });
+        },
+
+        "Should reset the user and provide the response when logging out": function () {
+            this._mockExpectations();
 
             this.app.logOut(function (err, resp) {
                 Y.Assert.areSame(
-                    logOutError, err, "The error of the logout should be provided"
+                    this.logOutError, err, "The error of the logout should be provided"
                 );
                 Y.Assert.areSame(
-                    logOutResponse, resp, "The response of the logout should be provided"
+                    this.logOutResponse, resp, "The response of the logout should be provided"
                 );
             });
 
             Y.Mock.verify(this.userMock);
             Y.Mock.verify(this.capiMock);
         },
+
+        "Should fire 'logOut' event when logging out": function () {
+            var logOutFired = false;
+
+            this._mockExpectations();
+            Y.Mock.expect(this.viewInfoMock, {
+                method: 'instance',
```

Comment:
that's completely wrong. The `viewInfo` is a plain JavaScript object with a property `instance`, not a method. So this is completely useless.

Actually, in this test, you are just testing that the `logOut` event is fired, so you don't need to initialize anything (except maybe that the `sideViews` is an empty object) to test that.